packages/utils/interactive-hooks/src/useKeyProps.ts

```diff
@@ -55,3 +55,6 @@ export const useKeyDownProps = noOp2;
  * @returns KeyPressProps: An object containing the correct platform specific props to  handle key press
  */
 export const useKeyProps = noOp2;
+
+/** Exposes the behavior of useKeyProps for the current platform as a boolean */
+export const preferKeyDownForKeyEvents = false;
```

Comment:
Where do you actually read and use this? In client code?

Reply:
That's the idea -- it could be used in the client code. I plan to handle my key + Enter + Space in the handler indicated by this boolean, but in theory, someone could do what I was previously doing, viz. use this flag to use the OTHER slot that isn't used by Fluent.